cumulusci.yml

```diff
@@ -1027,25 +1059,6 @@ flows:
                 options:
                     class_name: RD2_DataMigration_BATCH
 
-    enable_rd2_managed:
```

Comment:
The description (line 1008)for `enable_rd2` flow should be modified to indicate it works for both managed and un-managed code.